File: back/src/main/java/com/projet/valatMoslehGrodet/dto/DemandDTO.java
package com.projet.valatMoslehGrodet.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class DemandDTO {
    private Long id;
    private AccountDTO sender;
    private Boolean isAccepted;
    private Long eventId;

}

File: front/src/lib/api/models/DemandDto.ts
import { AccountDto } from "./AccountDto";

export type DemandDto = {
  /**
   * @type integer | undefined, int64
   */
  id?: number;
  sender?: AccountDto;
  /**
   * @type boolean | undefined
   */
  isAccepted?: boolean;
  /**
   * @type integer | undefined, int64
   */
  eventId?: number;
};
